### ISSUE TITLE
Mark homework 1 exercise 3 of Victoria Mansfield

### ISSUE THREAD

**Issue by vmansfield** [vmansfield/stat679work#2]
@cecileane 

[Link to Homework 1 Exercise 3](https://github.com/vmansfield/stat679work/commit/51e53dd6b554ec728bd22dff7f0c7e26255f67e5)

Above is the link to the commit with my submission for homework 1 exercise 3. The following files are part of the assignment and are in the 'hw1-snaqTimeTests' folder:
1. readme.md
2. normalizeFileNames.sh
3. summarizeSNaQres.csv
4. summarizeSNaQres.sh

Running the command 'bash summarizeSNaQres.sh' will generate the 'summarizeSNaQres.csv' file.

Corrections were made to the 'summarizeSNaQres.sh' file to fix issues from the first homework submission.

**Comment by cecileane** [comment]
Thanks Victoria! This is my way of asking Sheng (@anyezijue49) and Lan (@lan) to try your script and provide feedback on it.

**Comment by anyezijue49** [comment]
Hi Victoria,

Great work! I like the way you save the result of each question in a variable, that makes your code clear, easy to check and work pretty good. Your notes describing each steps in your code are very impressive and help me to understand your thoughts. That's wonderful.

However, there are still some mistakes concerned me.
The first one is that your work print out the root name of files in shell, which, in my opinion, is not necessary. 
The second one is that when I run your code in my local computer, there is nothing contained under the `Nfail` column in output cvs file whereas your submitted cvs file does have number under the `Nfail` column. I think the error may occur in `grep -o "max number of failed proposals = [[:digit:]]*" log/${root}.log | grep -o "[[:digit:]]*`. Maybe you should try `sed` after using `grep`.
In addition, It will be better to record more information on your readme.md file.

I appreciated your use of the concepts we've learned in class so far: loops, grep, and sed. The way you use `if` is quite skilled and cleared, I liked it.

Anyway, I thought your work was satisfactory.

Thanks for sharing your work! It was a pleasure to learn from your code.

Best,
Sheng

**Comment by cecileane** [comment]
To chime in just a tiny bit: I think it's okay for the script to print out something to the screen to show the progress, if only to say "the output was written to file xxx". Messages printed to the screen are not necessary, but they can be useful. They can be written in a such a way as to tell the user what's going on (or at least show that _something_ is going on).

**Comment by lan** [comment]
Hi Victoria,

I downloaded your stat679work repository in my computer and try to run it. I first went to the subdirectory where you put your summarizeSNaQres.sh and ran it. But there was an error saying there is `sed: log/*.log: No such file or directory`. So I went to the subdirectory where the `log` files were, and then used the code `sh ../../hw1-snaqTimeTests/summarizeSNaQres.sh`. This time, I successfully got the summarized result. Great!

I think your documentation is great. And the comments are clear enough. But I think the names of each subdirectory are not clear enough to me. I have noticed that there are two subdirectories named `hw1-snaqTimeTests` but in different directories and have different contents. I think maybe it is better to use `data`, `scripts`, `results` as the name of the folders (as the Professor suggested).

Overall, I think your work is great.
